src/collar/oc_capture.lsl

```diff
@@ -241,13 +436,15 @@ UserCommand(integer iNum, string sStr, key kID, integer remenu) {
 default{
 
     state_entry() {
-       //llSetMemoryLimit(32768); //2015-05-06 (4840 bytes free)
+        llSetMemoryLimit(32768); //2015-05-06 (4840 bytes free)
```

Comment:
You uncomment a memory restriction here after you added code... 
I hope you did severe testing here!